### PR TITLE
Create a post

### PR DESCRIPTION
**Pull request by mgjeffries**
# Description
Added a create method to the post view so that the client can create new posts. The publication date is automatically set by the database. The post is automatically set to be approved.
## Type of change
- [ ] New feature (non-breaking change which adds functionality)
# How Has This Been Tested?
Please describe step-by-step the process that a reviewer can follow to adequately test this PR. 
- [ ] send a POST request to `http://localhost:8000/posts` with some post data, such as as `{
    "title": "Test Post",
    "content": "LETS HIT IT AND QUITITTT",
    "image_url": "",
    "category_id": 1
}`
- [ ] Verify that a new post is returned, with an Id attached, and expanded by user and by category.
 - [ ] Change the category_id to a value that is not valid for your database, such as 300. Verify that a helpful message is displayed.
# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

File: rareapi/views/Posts.py [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
+from django.core.exceptions import ValidationError
 
 class PostViewSet(ViewSet):
+    """Rare Posts"""
+    def create(self, request):
+        """Handle POST operations
+
+        Returns:
+            Response -- JSON serialized game instance
+        """
```
Comment (Michaelr-Tyler) [inline rareapi/views/Posts.py:18]: Maybe change this doc string to post instance

File: rareapi/views/Posts.py [hunk 2]
```diff
@@ -1,12 +1,68 @@
+<<<<<<< HEAD
```
Comment (skratz17) [inline rareapi/views/Posts.py]: the aforementioned merge conflict remnants

**Review by Michaelr-Tyler — APPROVED** [review]
👍

**Review by skratz17 — APPROVED** [review]
Lookin good to me

**Pull request merged**